### PR TITLE
Isteelx ~~> MechSteelix + Minor Tweaks

### PR DESCRIPTION
**Pull request by Lycanium-Z**

### CODE REVIEW

File: mods/wlssb/moves.js [hunk 1]
```diff
@@ -429,7 +429,7 @@ exports.BattleMovedex = {
 		shortDesc: "10% chance to inflict dread on the target.",
 		id: "heartofdarkness",
 
-		name: "Ḫ̱̙̞̅̓͑͝e̲̅̿a̤͕̦͉ř̗̿ͫ̽͢t̼̝͊͒ͭ̀ ̖̦͆ͯỏͧf̻̙ ̺̏̂̉͐ͨ̐̃͟D̼̦͕̅ͬ̚a̝̻͉̰̯̓̊̇r͉͇̿̽ͤ̆k̍̑ͯ̃͟ͅn̺̞̤͉̪ͦ͒̒ͪ̄͑ě̴̖̙̩̘̂ŝ̸̇ͪ̃ͬŝ̨͙̙͚͎̬̂̒ͅ",
```
Comment (Lycanium-Z) [inline mods/wlssb/moves.js]: Dw about that

File: mods/wlssb/moves.js [hunk 2]
```diff
@@ -443,7 +443,7 @@ exports.BattleMovedex = {
 			this.add('-anim', source, "heart Stamp", target);
 		},
 		onHit: function (target) {
-			this.add('c|%Lycanium Z|T̡̞͚͈̥̋ͧͯ̈̓ͩ̆i̗̱̦ͦͦͅm̞̥̰͎̱ͬ̅̆̈̽e̢͕̞̙͆ͤͫ ̲̳̮ͩ̐͌̈̔͐͗͢t̗ͤ̂̃o̱̖̤̤̟͔͛̐ͩ̌ ͍͚̺͍̹̓ͨs̢̞̮̏ḫ͖̱̖̍́ͮͧ̚ò̷͔ͫw̨̞͕̻̓̒͋ͮ ̧ͣ̚y͈͚͉̥̳̒́̇͌̓̈̑ŏ̼͇̠̰̖̈́̅ͧ̓͗́u̴͇̟̔̆͆͂ ͔̖̼͔̣̗̲͋̆̊̈̊̓̊͢w̹̻͔̠̙̫ͫͧ͛̅̑h͚̣̳̮̪͌̉ͥ̀̀a̽͏̘̘͕̹t̤̟̝̘̺͉̼̽ ̥̻̳͖̮̅ͤ̏͌͗ͅi̼͍̍́͢t͉̙̳̹ͤ͝s̺͒ͨ̔̋̓͛ ̺͈̹̻̟̱̞̅ͩͦ͋̋̾͛lͯ҉̪͉̝̻̙i̗̺͖̞ͅk͗̎͑̅ͯ҉̭e̅̊ͥ̄̑ͭͩ ̸̩̜̼̀ͫ̿͗̃ͅt̛̠̑̍ͤͤͩͬ̚ǒ̲̜̼͕̞ ̋̾͛̃̎̾͡d̟̯̤͚̤̬̐͟įĕ̻̞̖͖͎̿͛!̞̳̘̳͒̇̋̈');
```
Comment (Lycanium-Z) [inline mods/wlssb/moves.js]: Dw bout that as well

File: mods/wlssb/statuses.js [hunk 3]
```diff
@@ -186,6 +186,7 @@ exports.BattleStatuses = {
 	lycaniumz: {
 		exists: true,
 		onStart: function (pokemon) {
+			pokemon.types = ["Dark"];
```
Comment (Lycanium-Z) [inline mods/wlssb/statuses.js]: "Why is this here?" You ask? Well apparently the aesthetics say Arceus-Dark but i found out that its actually arceus-normal. This makes sure its actually arceus-dark through and through.

File: wavelength-plugins/misc.js [hunk 4]
```diff
@@ -487,15 +487,15 @@ exports.commands = {
...
+		if (['km', 'callie', 'krakenmeme'].includes(toId(target))) return this.sendReplyBox(getMonData('krakenmare'));
 		if (toId(target) === toId('c7')) return this.sendReplyBox(getMonData('c733937123'));
 		if (['des', 'deso'].includes(toId(target))) return this.sendReplyBox(getMonData('desokoro'));
 		if (['mos', 'electricz'].includes(toId(target))) return this.sendReplyBox(getMonData('mosmero'));
 		if (toId(target) === toId('prince')) return this.sendReplyBox(getMonData('wavelengthprince'));
 		if (toId(target) === toId('hiro')) return this.sendReplyBox(getMonData('hiroz'));
 		if (['hh', 'hoeen'].includes(toId(target))) return this.sendReplyBox(getMonData('hoeenhero'));
-		if (toId(target) === toId('[]')) return this.sendReplyBox(getMonData('arrays'));
-		if (['mech', 'mechsteelix'].includes(toId(target))) return this.sendReplyBox(getMonData('isteelx'));
+		if (toId(target) === toId('arrays')) return this.sendReplyBox(getMonData('volco'));
```
Comment (HoeenCoder) [inline wavelength-plugins/misc.js]: why are we running toId on something that is an id?

Comment (Lycanium-Z) [reply wavelength-plugins/misc.js]: No idea tbh. I just edited stuff @HoeenCoder

Comment (HoeenCoder) [reply wavelength-plugins/misc.js]: edit the toId out then
